QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
```

Comment:
That wasn't correct thinking on my part.
Thinking about this more - we don't call this method directly, outside of this class - so whether it is static or not doesn't matter for what I was thinking before, I think I was a bit confused.

In this case, is there any advantage (time savings?) to having a method be static here?